python-package/mlbox/preprocessing/drift_thresholder.py

```diff
@@ -2,14 +2,14 @@
 # Author: Axel ARONIO DE ROMBLAY <axelderomblay@gmail.com>
 # License: BSD 3 clause
 
-from .drift import *
+from .drift import DriftThreshold
 import os
```

Comment:
@NCherel 

Usually module imports using only `import` are put at the beginning of the file and are then **followed** by all the `from module import method`.

You just need to invert the order 😉